### PR TITLE
add support for broadcom videocore IV vulkan driver

### PR DESCRIPTION
**Pull request by gravitone**

### CODE REVIEW

**Comment by JosJuice** [comment]
This change doesn't actually do anything on its own.

**Comment by JosJuice** [comment]
Ah, is PR #10554 meant to be combined with this PR? You'll have to submit the the two changes in one PR, otherwise it doesn't really make sense.

**Comment by gravitone** [comment]
apologies, first time actually using git. Both files are now included in this commit.

File: Source/Core/VideoBackends/Vulkan/VulkanContext.cpp [hunk 1]
```diff
@@ -922,6 +922,12 @@ void VulkanContext::InitDriverDetails()
     vendor = DriverDetails::VENDOR_IMGTEC;
     driver = DriverDetails::DRIVER_IMGTEC;
   }
+  else if (vendor_id == 0x14E4 || device_name.find("V3D 4.2") != std::string::npos)
+  {
+    // support for the Broadcom Videocore IV.
```
Comment (JosJuice) [inline Source/Core/VideoBackends/Vulkan/VulkanContext.cpp:927]: This comment is unnecessary.

**Review by JosJuice — COMMENTED** [review]
That's better, but currently you have two commits in this pull request, and we would prefer if there was just one. Please squash the two commits into one, then force push.

**Comment by gravitone** [comment]
Can I do this in the existing commit or should I close it and a create a new one?

File: Source/Core/VideoBackends/Vulkan/VulkanContext.cpp (hunk 1 above)
Comment (gravitone) [reply Source/Core/VideoBackends/Vulkan/VulkanContext.cpp]: comment has been removed

**Comment by JosJuice** [comment]
You can do it in the existing pull request. (Commits and pull requests are separate things – commits can't be closed but pull requests can be.)

**Pull request closed without merging**